src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random rnd = new Random();
+    private static final int MAX_VALUE = 20;
+
+    public Figure getRandomFigure() {
+        int numFigure = rnd.nextInt(5);
```

Comment:
lets move 5 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names